Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for cBioPortal
+FROM elementolab/jamm:1.0
+MAINTAINER Alexandros Sigaras <als2076@med.cornell.edu>
+#===========================#
+# Clone cBioPortal			#
+#===========================#
+#RUN git clone https://github.com/cBioPortal/cbioportal.git; \
+ADD / /cbioportal
+RUN	echo "export PORTAL_HOME=/cbioportal" >> /root/.bash_profile;
+#===========================#
+# Setup MySQL				#
+#===========================#
+RUN chkconfig mysqld on; \
+	service mysqld start; \
+	mysql_secret=$(cat /var/log/mysqld.log | grep "A temporary password is generated for" | awk '{print $NF}'); \
+	new_pass="P@ssword1"; \
+	cbio_user="cbio"; \
+	cbio_pass="P@ssword1"; \
+	mysql -u root -p$mysql_secret --connect-expired-password -e "set password=password('$new_pass'); flush privileges;"; \
+	mysql -u root -p$new_pass -e "create database cbioportal; create database cgds_test; CREATE USER '$cbio_user'@'localhost' IDENTIFIED BY '$cbio_pass'; GRANT ALL ON cbioportal.* TO '$cbio_user'@'localhost'; GRANT ALL ON cgds_test.* TO '$cbio_user'@'localhost'; flush privileges;" ;
```

Comment:
Is this step really necessary to build cBioPortal, without running integration tests? Since (I believe) we’re provisioning an a filesystem and process environment to run either 1) a Tomcat server running cBioPortal, 2) a Git pull and Maven build of cBioPortal or 3) the cBioPortal import scripts, I think we can be a lot more efficient by not including MySQL in it at all.